src/simulate.js

```diff
@@ -28,6 +29,14 @@ export const render = (app: FusionApp) => (
   url: string,
   options: * = {}
 ): Promise<*> => {
+  if (global.jsdom) {
+    if (!url.startsWith('/')) {
+      url = `/${url}`;
+    }
+    global.jsdom.reconfigure({
+      url: `http://localhost${url}`,
```

Comment:
How does this work with ports?

Reply:
It shouldn't matter what the host or port is, just the path

Reply:
With JSDOM there isn't actually a port, so yeah, I think it's fine.